docs/Changelog.md

```diff
@@ -18587,6 +18587,95 @@ This version of the operator has been available since version 13 of the default
 </dl>
 
 ## Version 14 of the default ONNX operator set
+### <a name="BatchNormalization-14"></a>**BatchNormalization-14**</a>
+
+  Carries out batch normalization as described in the paper
+  https://arxiv.org/abs/1502.03167. Depending on the mode it is being run,
+  There are five required inputs 'X', 'scale', 'B', 'input_mean' and
+  'input_var', in addition to one optional input 'training_mode'.
+  Note that 'input_mean' and 'input_var' are expected to be the estimated
+  statistics in inference mode (training_mode=False, default),
+  and the running statistics in training mode (training_mode=True).
+  There are multiple cases for the number of outputs, which we list below:
+  
+  Output case #1: Y, running_mean, running_var, current_mean, current_var (training_mode=True)
+  Output case #2: Y (training_mode=False)
+  
+  When training_mode=False, extra outputs are undefined and the user should not depend on those.
+  The outputs are updated as follows when training_mode=True:
+  ```
+  current_mean = ReducedMean(X, axis=all_except_channel_index)
+  current_var =  ReducedVar(X, axis=all_except_channel_index)
+  
+  running_mean = mean * momentum + current_mean * (1 - momentum)
```

Comment:
Sorry - still not sure if this should be `input_mean` and `input_var` instead of `mean` and `var` ?